react/Autosuggest/Autosuggest.demo.js

```diff
@@ -49,6 +49,15 @@ class AutosuggestContainer extends Component {
           }}
           onClear={this.handleClear}
         />
+        <DemoComponent
```

Comment:
The 2nd demo component has the same value as the first one - would it be possible for them to be independent?